Dockerfile

```
@@ -9,31 +10,43 @@ RUN groupadd --gid ${USER_ID} ${GROUP_ID} && \
     useradd --create-home --uid ${USER_ID} --gid ${GROUP_ID} --home-dir /app ${GROUP_ID}
 
 # For grpc https://github.com/grpc/grpc/issues/24556#issuecomment-751797589
-RUN apt-get update -qqy && \
-    apt-get install -qqy python-dev build-essential git curl software-properties-common
-
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-key C99B11DEB97541F0
-RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key C99B11DEB97541F0
-RUN apt-add-repository https://cli.github.com/packages
-RUN apt update
-RUN apt install -y gh
+RUN apt-get update -qqy \
+    && apt-get install --no-install-recommends -qqy \
+        # python-dev \
+        # build-essential \
+        # curl \
```

I think we still need those, for example for this: https://github.com/mozilla/lookml-generator/blob/df244e9a545a057776f15d277e63db8f7d9a1172/bin/generate#L235
```suggestion
        python-dev \
        build-essential \
        curl \
```

---

do we need `python-dev` and `build-essential` though? Is there any way for us to test this?

---

Turns out they are used for `ssh-keyscan`. Adding them back